libs/core/splitter/splitter-split-pane/splitter-split-pane.component.ts

```diff
@@ -38,7 +38,10 @@ export class SplitterSplitPaneComponent implements OnInit, AfterViewInit, OnDest
     @Input()
     set size(size: string) {
         this._size = size;
-        this._actualSize = this._size;
+        if (this._actualSize !== this._size) {
+            this._actualSize = this._size;
+            this.sizeChange.emit();
+        }
```

Comment:
I would rewrite it with signals, and in `splitter-pane-container` I would use effect that reacts on every pane.size$() update